templates/python/package/aio/services/service.py.twig

```diff
@@ -0,0 +1,83 @@
+from ..service import Service
+from ..exception import AppwriteException
+
+class {{ service.name | caseUcfirst }}(Service):
```

Comment:
Could we use the existing service as a base class and extend here as a child `AsyncService`? Then we would only need to maintain the async methods in the `aio` package